Two symbols missing in simulator setup

Hello,

SecKeychainGetTypeID and SecAccessGetTypeID are missing at link time when I target the iphone6 simulator. Strangely I don't have the problem when I target an actual iphone6. Do we have another cfg() mixup somewhere ?

```
undef: _SecKeychainGetTypeID
undef: _SecAccessGetTypeID
Undefined symbols for architecture x86_64:
  "_SecKeychainGetTypeID", referenced from:
      _$LT$security_framework..keychain..SecKeychain$u20$as$u20$core_foundation..base..TCFType$LT$$BP$mut$u20$security_framework_sys..base..OpaqueSecKeychainRef$GT$$GT$::type_id::h0b49430b489586d5 in SnipsSda(security_framework-d4c2fbca6aed6c3a.0.o)
  "_SecAccessGetTypeID", referenced from:
      _$LT$security_framework..access..SecAccess$u20$as$u20$core_foundation..base..TCFType$LT$$BP$mut$u20$security_framework_sys..base..OpaqueSecAccessRef$GT$$GT$::type_id::h5bfb1124c8dad652 in SnipsSda(security_framework-d4c2fbca6aed6c3a.0.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/security-framework-sys/src/lib.rs
+++ b/security-framework-sys/src/lib.rs
@@ -4,6 +4,7 @@
 extern crate core_foundation_sys;
 extern crate libc;
 
+#[cfg(target_os = "macos")]
 pub mod access;
 pub mod base;
 pub mod certificate;
@@ -16,6 +17,7 @@ pub mod identity;
 pub mod import_export;
 pub mod item;
 pub mod key;
+#[cfg(target_os = "macos")]
 pub mod keychain;
 #[cfg(target_os = "macos")]
 pub mod keychain_item;

--- a/security-framework/src/import_export.rs
+++ b/security-framework/src/import_export.rs
@@ -8,7 +8,9 @@ use core_foundation::dictionary::CFDictionary;
 use core_foundation::array::CFArray;
 use std::ptr;
 
+#[cfg(target_os = "macos")]
 use access::SecAccess;
+#[cfg(target_os = "macos")]
 use keychain::SecKeychain;
 use trust::SecTrust;
 use certificate::SecCertificate;

--- a/security-framework/src/item.rs
+++ b/security-framework/src/item.rs
@@ -16,6 +16,7 @@ use certificate::SecCertificate;
 use cvt;
 use identity::SecIdentity;
 use key::SecKey;
+#[cfg(target_os = "macos")]
 use keychain::SecKeychain;
 
 /// Specifies the type of items to search for.
@@ -70,6 +71,7 @@ impl ItemSearchOptions {
     /// Search within the specified keychains.
     ///
     /// If this is not called, the default keychain will be searched.
+    #[cfg(target_os = "macos")]
     pub fn keychains(&mut self, keychains: &[SecKeychain]) -> &mut ItemSearchOptions {
         self.keychains = Some(CFArray::from_CFTypes(keychains));
         self

--- a/security-framework/src/lib.rs
+++ b/security-framework/src/lib.rs
@@ -23,9 +23,11 @@ use core_foundation_sys::base::OSStatus;
 use security_framework_sys::base::errSecSuccess;
 use security_framework_sys::cipher_suite::SSLCipherSuite;
 
+#[cfg(target_os = "macos")]
 use access::SecAccess;
 use base::{Result, Error};
 use cipher_suite::CipherSuite;
+#[cfg(target_os = "macos")]
 use keychain::SecKeychain;
 
 macro_rules! make_wrapper {
@@ -65,6 +67,7 @@ macro_rules! p {
     }
 }
 
+#[cfg(target_os = "macos")]
 pub mod access;
 pub mod base;
 pub mod certificate;
@@ -73,6 +76,7 @@ pub mod identity;
 pub mod import_export;
 pub mod item;
 pub mod key;
+#[cfg(target_os = "macos")]
 pub mod keychain;
 pub mod os;
 pub mod policy;

